lib/getResourceIdentifier/testResources.js

```diff
@@ -0,0 +1,120 @@
+const eresourceOne = {
```

Comment:
I'm aware this is the pattern we have at the minute in ERM, but I'm really not a fan of it.


I would prefer that we have a set of exports from the `test/jest` directory which we can import so that we have a consistent set of data to draw from in our tests, rather than setting up `eresource` in 5 places. If our API changes then we have a LOT of places to change this at the moment, instead of one, and it will lead to us missing regressions in the tests.


Please can you set up a file named something reflecting which endpoint it's "mocking" in `test/jest`, and export that/import it into your test as a list, like we'd get back from the endoint?

Reply:
I'm afraid I don't undestand what you mean.